src/Pages/MembershipApplication/MembershipForm.js

```diff
@@ -73,6 +74,31 @@ export default function MembershipForm (props) {
       })
   }
 
+  async function sendVerificationEmail () {
+    await axios
+      .post('/api/mailer', {
+        templateType: 'verification',
+        recipientEmail: email,
+        recipientName: firstName
+      })
+      .then(async result => {
+        setEmailToVerified()
+      })
+      .catch(error => {
+        console.log(error)
+      })
+  }
+
+  async function setEmailToVerified () {
```

Comment:
Also abstract this axios call. #151 describes the abstraciton effort

Reply:
Completed in the newest push